cmd/ocitools/validate.go

```diff
@@ -340,6 +336,14 @@ func checkLinux(spec rspec.Spec, rootfs string, hostCheck bool) (msgs []string)
 		}
 	}
 
+	if (len(spec.Linux.UIDMappings) > 0 || len(spec.Linux.GIDMappings) > 0) && !userExists {
+		msgs = append(msgs, "UID/GID mappings require User namespace exists")
```

Comment:
Thanks, got it. 
Since spec does not allow now, I fixed error message as you suggested.